docs/src/main/asciidoc/logging.adoc

```diff
@@ -416,8 +412,8 @@ For details about its configuration, see the xref:#quarkus-log-logging-log-confi
 
 === File log handler
 
-By default, the file log handler in Quarkus is disabled.
-Once enabled, it enables the logging of all events to a file on the application's host, while also supporting log file rotation.
+The Quarkus file log handler is disabled by default, but when enabled, it logs events to a file on the host, supporting log file rotation.
```

Comment:
Or, you could also describe it from the user's perspective/goal:

```suggestion
To log events to a file on the host, use the Quarkus file log handler, which is disabled by default.

The Quarkus file log handler supports log file rotation. 
Log file rotation ensures effective log file management over time by maintaining a specified number of backup log files, while also keeping the primary log file up-to-date and manageable.
```

Side note: :grey_question:  Wondering why this Quarkus feature wasn't called _Log file handler_.